config/config_soft_filer_somatic.yaml

```diff
@@ -0,0 +1,21 @@
+filters:
+  depth:
+    description: "Soft filter on depth"
+    expression: "FORMAT:DP:200"
+    soft_filter: "True"
+  ad:
+    description: "Soft filter variants with few observations (AD lower than 20)"
+    expression: "(FORMAT:AD:1 < 20)"
```

Comment:
If depth target is 1000x this would filter out anything under 2%. Should it be lower?